src/javascript/app/pages/user/view_popup/view_popup.js

```diff
@@ -56,6 +56,11 @@ const ViewPopup = (() => {
         }
 
         getContract();
+
```

Comment:
there is a same if statement above, why we need to disable it and then enable it again? doesn't removing above statement fix the issue?

Reply:
True also, but it disables the button at the back of the statement. Then it should enable back with the cleanup.